src/psyclone/tests/psyGen_test.py

```diff
@@ -3691,6 +3754,45 @@ def test_fparser2astprocessor_handling_binaryopbase(f2008_parser):
     assert new_node._operator == '+'
 
 
+def test_fparser2astprocessor_handling_unaryopbase(f2008_parser):
+    ''' Test that fparser2 UnaryOpBase is converted to the expected PSyIR
+    tree structure.
+    '''
+    from fparser.common.readfortran import FortranStringReader
+    from fparser.two.Fortran2003 import Execution_Part
+    reader = FortranStringReader("x=-4")
+    fparser2unary_operation = Execution_Part.match(reader)[0][0].items[2]
+
```

Comment:
Please add a check here that `fparser2unary_operation` is what you expect it to be.

Reply:
done